src/torch/12-deeprank-train/deeprank-train.lua

```diff
@@ -26,7 +27,7 @@ function getSimilarityModel()
   --leftImage:add(nn.SpatialConvolutionMM(512, 126, 1, 1, 1, 1, 0, 0):cuda())
```

Comment:
Not sure, my eyes skip them when I am not interested in this part of the code, but it does add value when you are looking for things that have been tried in the past.

Your call